extension/cgroupruntimeextension/CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing to the Cgroup Go runtime extension
+
+In order to contribute to this extension, it might be useful to have a working local setup.
+
+## Testing
+
+To run the integration tests locally for this extension, you can follow theses steps in a Linux environment.
```

Comment:
Note that some Linux distributions already run systemd under cgroupv2, in that case, integration tests can be run without the docker workaround.

Reply:
Notice added https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/36920/commits/12acc97e2433ec7a99ce64b5bd26628bea405752